src/test/java/com/apache/hadoop/hive/ql/parse/ParseDriverTest.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.apache.hadoop.hive.ql.parse;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.LinkedList;
+
+import org.apache.hadoop.hive.ql.parse.ASTNode;
+import org.apache.hadoop.hive.ql.parse.CalcitePlanner;
+import org.apache.hadoop.hive.ql.parse.CalcitePlanner.ASTSearcher;
+import org.apache.hadoop.hive.ql.parse.HiveParser;
+import org.apache.hadoop.hive.ql.parse.MutantSwarmParseDriver;
+import org.apache.hadoop.hive.ql.parse.ParseException;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.stubbing.OngoingStubbing;
+
+public class ParseDriverTest {
+
+  private ASTNode tree;
+
+  private ASTNode node;
+
+  private String command;
+
+  @Before
+  public void setUpMocks() {
+    command = "CREATE TABLE foobar AS\n" + 
+        "SELECT c\n" + 
+        "from bar\n" + 
```

Comment:
I'd try and be consistent with the case for the reserved words, for instance since you have `CREATE TABLE` and `SELECT` in uppercase I'd do the same with `FROM` and `WHERE`.